src/shadowbox/server/manager_metrics.ts

```diff
@@ -40,4 +56,21 @@ export class PrometheusManagerMetrics implements ManagerMetrics {
     }
     return {bytesTransferredByUserId: usage};
   }
+
+  async getTunnelTimeByLocation({
+    time_window: {seconds},
```

Comment:
To ensure it's the proper type here
```suggestion
    time_window: Duration,
```